### PR TITLE
[Sprint 11 / Feature] Pseudo Label Script Utilites

### PR DESCRIPTION
**Pull request by Nathan24zz**
## Jira Link: 
https://ichiro-its.atlassian.net/browse/PI-178

## Description
Add argument handler in pseudo_label.py. 
Add another utility like delete related files and text when the detection result is empty and change annotation format from `(class_id, x, y, width, height)` to `(class_id, center_x, center_y, width, height)`

## Type of Change

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause the existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] New unit tests added.
- [x] Manual tested.

## Checklist:

- [x] Using Branch Name Convention
    - `feature/JIRA-ID-SHORT-DESCRIPTION` if has a JIRA ticket
    - `enhancement/SHORT-DESCRIPTION` if has/has no JIRA ticket and contain enhancement
    - `hotfix/SHORT-DESCRIPTION` if the change doesn't need to be tested (urgent)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made the documentation for the corresponding changes.

### CODE REVIEW

File: change_annotation_format.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# THE SOFTWARE.
+
+import argparse
+import os
+from tqdm import tqdm
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--data_path', help='path for image data',
+                      type=str, default="data_coba")
```
Comment (segara2410) [inline change_annotation_format.py]: better to use "data" for the default

File: pseudo_label.py [hunk 2]
```diff
@@ -1,21 +1,61 @@
...
-  config = os.path.join(data_path, "config.cfg")
-  labels = os.path.join(data_path, "obj.names")
-  draw_output = True
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--CONF_THRESH', help='threshold for object confident',
+                      type=float, default=0.4)
+  parser.add_argument('--config', help='path for yolo config',
+                      type=str, default="data/config.cfg")
+  parser.add_argument('--CUDA', help='threshold for Non-maximum Suppression',
+                      type=bool, default=False)
```
Comment (segara2410) [inline pseudo_label.py]: I don't think you write a correct help here

File: delete_empty_detection.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  # loop every directory in <data_path> directory
+  for name in os.listdir(data_path):
+    target_dir = os.path.join(data_path, name)
+    if os.path.isdir(target_dir):
+      print(f"\nRunning on directory {target_dir}...")
+
+      with open(f"{target_dir}.txt", "r") as file_list_text:
+
+        for everything in tqdm(os.listdir(target_dir)):
+          full_path = os.path.join(target_dir, everything)
```
Comment (segara2410) [inline delete_empty_detection.py]: I think it's better to use `file` instead of `everything`

File: pseudo_label.py [hunk 4]
```diff
@@ -1,21 +1,61 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import argparse
+from ctypes.wintypes import BOOL
```
Comment (segara2410) [inline pseudo_label.py]: what is this for?

File: change_annotation_format.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            w = float(w)
+            h = float(h)
+            x = float(center_x) - w/2
+            y = float(center_y) - h/2
+            detected_objects[i] = f"{class_id} {x} {y} {w} {h}\n"
+          
+          with open(f"{file_name}.txt", "w") as file:
+            file.writelines(detected_objects)
+
+
```
Comment (segara2410) [inline change_annotation_format.py]: extra whitspaces?

**Comment by segara2410** [comment]
I think this one count as a feature PR, please update the title and complete the desc with the features that added here

File: change_annotation_format.py (hunk 1 above)
Comment (Nathan24zz) [reply change_annotation_format.py]: I forgot to change it after testing hehehe :)

File: pseudo_label.py (hunk 2 above)
Comment (Nathan24zz) [reply pseudo_label.py]: ya I will fix it

File: delete_empty_detection.py (hunk 3 above)
Comment (Nathan24zz) [reply delete_empty_detection.py]: ok

File: change_annotation_format.py (hunk 5 above)
Comment (Nathan24zz) [reply change_annotation_format.py]: ok I will delete it

File: pseudo_label.py (hunk 4 above)
Comment (Nathan24zz) [reply pseudo_label.py]: I think that is not required

**Review by segara2410 — APPROVED** [review]

**Pull request merged**